src/Event.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Beaver\Dateval;
```

Comment:
__[code_sniffer]__
* There must be one blank line after the namespace declaration